examples/gitlab-centos7-atomicapp/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM projectatomic/atomicapp:0.1.3
+
+MAINTAINER Navid Shaikh <nshaikh@redhat.com>
+
+LABEL io.projectatomic.nulecule.specversion="0.0.2" \
+      io.projectatomic.nulecule.providers="kubernetes"\
+      Build="docker build --rm --tag swordphilic/gitlab-atomicapp ."
+
+ADD /Nulecule /Dockerfile README.md gpl-3.0.txt answers.conf.sample /application-entity/
```

Comment:
I wanted to provide a sample `answer.conf.sample` so that user can view and relate, however I could find the same `answer.conf.sample` after `install` - it generates a minimal answer file with provider and namespace fields in general section. is this expected? how can package the `answer.conf.sample` file so that after `install` atomicapp can populate it for me in $pwd ?